controllers/ptpconfig_controller.go

```diff
@@ -47,8 +45,7 @@ type PtpConfigReconciler struct {
 //+kubebuilder:rbac:groups=config.openshift.io,resources=infrastructures,verbs=get;list;watch
 
 func (r *PtpConfigReconciler) Reconcile(ctx context.Context, req ctrl.Request) (reconcile.Result, error) {
-	reqLogger := r.Log.WithValues("Request.Namespace", req.Namespace, "Request.Name", req.Name)
-	reqLogger.Info("Reconciling PtpConfig")
+	glog.Infof("Request.Namespace %s Request.Name %s - Reconciling PtpConfig", req.Namespace, req.Name)
```

Comment:
The interface that Operator SDK (and controller-runtime) use for logging is actually logr, so any logging implementation that supports the logr interface can be **used.
These** are defaults used by operator-sdk.  By changing these there could be issues. We need to test these changes. Also think about  operator-sdk upgrades

Reply:
Interesting, this makes sense.  We can disregard this then.  Thanks!